tasks.py.jinja

```diff
@@ -292,6 +344,59 @@ def start(c, detach=True, debugpy=False):
         c.run(cmd, env=dict(UID_ENV, DOODBA_DEBUGPY_ENABLE=str(int(debugpy))))
 
 
+@task(
+    develop,
+    help={
+        "debugpy": "Whether or not to run tests in a VSCode debugging session. " \
+            "Default: False",
+        "modules": "Comma-separated list of modules to test. Default: 'base'.",
+        "logs": "Whether or not to follow Odoo logs during testing. " \
+            "If True, Odoo will also be automatically restarted when tests finish. " \
+            "Default: True",
+    },
+)
+def test(c, modules="base", debugpy=False, logs=True):
+    """Run Odoo tests"""
+    tmp_docker_compose_file = tempfile.NamedTemporaryFile(
+        dir=PROJECT_ROOT, mode="w", suffix=".yaml"
+    )
+    cmd = "docker-compose -f docker-compose.yml " \
+        f"-f {tmp_docker_compose_file.name} up -d"
+    odoo_command = f"odoo --test-enable -i {modules} --stop-after-init"
+    _override_docker_command(
+        c, "odoo", odoo_command, file=tmp_docker_compose_file
+    )
+    with c.cd(str(PROJECT_ROOT)):
+        c.run(
+            cmd,
+            env=dict(
+                UID_ENV,
+                DOODBA_DEBUGPY_ENABLE=str(int(debugpy)),
+            ),
+        )
+        if logs:
+            c.run(
+                "docker-compose logs -f odoo",
+                env=dict(
+                    UID_ENV,
+                ),
+            )
```

Comment:
I ended up with this approach because, if we run `docker-compose` in detached mode, we don't get the test logs and we don't know when the command ends.
If we run it it normal mode, we get the logs from all the services, and still don't know when the tests finished. 
This allows us to get both.